gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -295,13 +309,18 @@ The remaining procedures walk you through steps for creating and testing a new H
 +
 image::gateway-quickstart-k8-api-config-endpoint.png["Screenshot of 'Endpoint' page",width=90%]
 +
+.. For the *Scheme*, use the default, *HTTP* setting.
++
+This guide focuses on a simple HTTP API. For HTTPS configuration details, see xref:flex-conn-tls-config.adoc[].
 .. Copy the following URI into the *Implementation URI* field:
 +
 [source]
 ----
 https://jsonplaceholder.typicode.com/
 ----
 .. Type `80` in the *Port* field.
++
+The example in this guide assumes an external IP that uses port `80` instead of the default port, `8081`. 
```

Comment:
I'm not sure what the external IP refers to. Also, 8081 is the default port in APIM, but 80 is the default for the HTTP protocol, therefore I don't think we can say that 8081 is "the default port".

Reply:
done